static/redfish/v1/schema/OemAssembly_v1.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+    <edmx:Reference Uri="http://docs.oasis-open.org/odata/odata/v4.0/errata03/csd01/complete/vocabularies/Org.OData.Core.V1.xml">
+        <edmx:Include Namespace="Org.OData.Core.V1" Alias="OData"/>
+    </edmx:Reference>
+    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/Assembly_v1.xml">
+        <edmx:Include Namespace="Assembly"/>
+        <edmx:Include Namespace="Assembly.v1_0_0"/>
+    </edmx:Reference>
+    <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
+        <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
+    </edmx:Reference>
+
+    <edmx:DataServices>
+
+        <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OemAssembly">
+            <Annotation Term="Redfish.OwningEntity" String="IBM"/>
```

Comment:
above you are saying OpenBMC 
                     aResp->res.jsonValue["Oem"]["OpenBMC"]["ReadyToRemove"] = true;
                
  maybe just change this to OpenBMC ?

Reply:
Here, in the relevant block of code, we are setting the value of ReadyToRemove to either "true" or "false" to return to the user in the bmcweb GET route.  Please specify what should be changed to "OpenBMC".